README.md

```diff
@@ -104,6 +104,9 @@ The following CMake options can be set (defaults are upper case):
     -DBUILD_FOR_SSE2=off|ON     build with SSE2 optimisaiton
     -DBUILD_DOC=OFF|on          build documentation
 
+    -DCMAKE_CXX_CLANG_TIDY=clang-tidy       run clang-tidy automatically, alongside with a regular compiler
```

Comment:
Because of `WarningsAsErrors: '*'` the build fails with this. I don't see an easy workaround, so I guess we have to live with it. Checking only specific files seems more useful anyway.

Reply:
I'll add explicit clang-tidy version and it won't break anything with our current configs. (with -misc-const-correctness)